src/components/shared/modals/modalWithInput/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useEffect} from 'react';
```

Comment:
We're no using a useEffect in this logic, please remove it

Reply:
done